openpmd_viewer/openpmd_timeseries/main.py

```diff
@@ -51,32 +45,32 @@ def __init__(self, path_to_dir, check_all_files=True):
         ----------
         path_to_dir: string
             The path to the directory where the openPMD files are.
-            For the moment, only HDF5 files are supported. There should be
-            one file per iteration, and the name of the files should end
-            with the iteration number, followed by '.h5' (e.g. data0005000.h5)
 
         check_all_files: bool, optional
             Check that all the files in the timeseries are consistent
             (i.e. that they contain the same fields and particles,
             with the same metadata)
             For fast access to the files, this can be changed to False.
         """
-        # Extract the files and the iterations
-        self.h5_files, self.iterations = list_h5_files(path_to_dir)
```

Comment:
We do not store `self.h5_files` anymore, in the `OpenPMDTimeSeries`, in order to be agnostic of the backend at this level.

But `self.iterations` is still stored (see below)